_posts/en/2020-12-04-better-tech-through-micro-procurementpt2.md

```diff
@@ -0,0 +1,53 @@
+Like all policy, though, procurement policy is open to interpretation and because of this, appetite for risk (by that procurement officer, procurement team, CFO or Deputy Head) can be a factor in implementing any procurement innovation.
+It can be seen as less risky to do things the way they have always been done.
+Related, a "fear of audit" was noted in the [Internal Red Tape Reduction report](https://internal-red-tape-reduction-report.github.io/chapter-3/#root-causes) as a driver of "the need to document every step of the process, which, in turn, drives the heavy paperwork, all in paper format."
+But as the Office of the Procurement Ombudsman pointed out in their most recent [annual report](http://opo-boa.gc.ca/rapports-reports/2019-2020/index-eng.html), simplifying and streamlining procurement in the GC "may require a higher degree of risk tolerance.”
+
+There is another challenge related to procurement policy.
+We know that flexibility exists within GC procurement policy to establish a simplified process for LDV procurements.
+However, this flexibility exists because policy doesn't explicitly say it cannot be done.
+As a result, there is very little guidance on how to do micro-procurement within the policy documents or at the departmental level.
+The Office of the Procurement Ombudsman noted this in their [2019 Low Dollar Value Report](http://opo-boa.gc.ca/faiblevaleur-dollarvalue-eng.html) stating that "The audit found that despite adopting the Government of Canada’s contracting policies, TBS's internal process was not supported by a sufficiently articulated policy framework specific to LDV contracting "particularly for non-expert contracting authorities."
+
+Departmental procurement rules can also create barriers to establishing micro-procurement initiatives in the GC.
+Unlike policy, departmental procurement rules can vary widely from department to department.
+This is especially true of rules around credit card use.
+Policy doesn't dictate things like credit card limits and individual transaction limits, but departmental rules do.
+When looking into establishing a micro-procurement initiative in the GC, a good first step would be to learn your department's rules around credit card use.
+
+On the IT side, in order for micro-procurement of digital work to be possible, we must break our work out into small chunks.
+This creates a challenge because we aren't used to working like that.
+We have been grouping IT tasks together into large IT projects for many years.
+ESDC's draft [Target Solution Delivery Model](https://sara-sabr.github.io/ITStrategy/strategy-target-solution-delivery-model.html) suggests that "In order to better deliver services to Canadians, we need to improve IT’s responsiveness, otherwise the feedback loop between policy makers and Canadians will be too long, affecting ESDC’s ability to iterate on its policy making and become an agile organization."
```

Comment:
Again, the exact URL (However, it scrolls a bit too low. I'll look into this, we had this issue in the past.):
https://sara-sabr.github.io/ITStrategy/strategy-target-solution-delivery-model.html#business-case-strategic-context

Reply:
Fixed!